### PR TITLE
Registration and Login Page

### PR DESCRIPTION
**Pull request by shovaaa**
1) Registration page - This is the main index page. The user will register from here and their data will be stored in the database named users in the table named register. Once the registration is completed, the user will be redirected to login page.
2) Login page - The user can login from here. If the user is registered then s/he will be redirected to the chat page and if not s/he will be redirected to the registration page.
3) Chat page - Currently in this update, the usernames are displayed in this page.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+server.set("view-engine", "ejs")
+//server.use(express.urlencoded({ extended: false}));
+
+server.get("/", (req, res)=>{
+    res.render("index.ejs")
+})
+
+server.post("/", urlencodedParser, (req, res)=>{
```
Comment (ojhaujjwal) [inline server.js:28]: please use `urlencodedParser` globally and not at each route. this is very tedious.

**Review by ojhaujjwal — CHANGES_REQUESTED** [review]

File: server.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    host: 'localhost',
+    user: 'root',
+    password: '',
+    database: 'users'
+})
+
+connection.connect(function(err){
+    if(err) throw err
+        console.log("You're connected.")  
+})
```
Comment (ojhaujjwal) [inline server.js:17]: what is request comes in before database connection is established?

Comment (ojhaujjwal) [inline server.js:17]: what is request comes in before database connection is established?

Comment (shovaaa) [reply server.js:17]: sorry but I didn't understand you. Can you please paraphrase it.

File: server.js (hunk 1 above)
Comment (shovaaa) [reply server.js:28]: okay. thanks for the feedback

**Pull request closed without merging**